sys/cam/cam_ccb.h

```diff
@@ -1438,7 +1439,7 @@ union ccb {
 	struct	ccb_async		casync;
 	struct	ccb_nvmeio		nvmeio;
 	struct	ccb_mmcio		mmcio;
-};
+} __no_subobject_bounds;
```

Comment:
Would probably be nice to drop this one eventually. I guess it means code takes a pointer to one type and then casts it to something else?

Reply:
Yeah, IIRC the issue is that the ccb is moved as a pointer to the ccb_hdr and then cast back to a `ccb *` that is then used to access members of other types depending on flags. We may be able to slightly modify the whole cam subsystem to always move `struct ccb *` values so it never has to do this.

Reply:
If it's just the header can we not annotate that member instead?

Reply:
I believe it is only the header that is used in this way. I moved the annotation.